src/views/GRTSystemIntakeReview/index.tsx

```diff
@@ -17,30 +27,160 @@ export const GRTSystemIntakeReview = () => {
+                }}
+              >
+                {(formikProps: FormikProps<SystemIntakeForm>) => {
+                  const { values, setFieldValue } = formikProps;
+                  return (
+                    <Form>
+                      <h1 className="font-heading-xl margin-top-0">
+                        Next steps
+                      </h1>
+                      <div className="grid-row">
+                        <div className="grid-col-4">
+                          <FieldGroup>
+                            <fieldset className="usa-fieldset">
+                              <legend className="usa-label margin-bottom-2">
+                                How to proceed?
+                              </legend>
+                              <Field
+                                as={RadioField}
+                                checked={values.status === 'APPROVED'}
+                                id="GrtIntakeReviewForm-Approved"
+                                name="decision"
+                                label="Issue Lifecycle ID with no further governance"
+                                onChange={() => {
+                                  setFieldValue('status', 'APPROVED');
+                                }}
+                                value
+                              />
+
+                              <Field
+                                as={RadioField}
+                                checked={values.status === 'ACCEPTED'}
+                                id="GrtIntakeReviewForm-Accepted"
+                                name="decision"
+                                label="Governance review process needed"
+                                onChange={() => {
+                                  setFieldValue('status', 'ACCEPTED');
+                                }}
+                                value
+                              />
+
+                              <Field
+                                as={RadioField}
+                                checked={values.status === 'CLOSED'}
+                                id="GrtIntakeReviewForm-Closed"
+                                name="decision"
+                                label="Governance not needed (close this request)"
+                                onChange={() => {
+                                  setFieldValue('status', 'CLOSED');
+                                }}
+                                value
+                              />
+                            </fieldset>
+                          </FieldGroup>
+                          <hr className="border-black border-bottom-0" />
+                          <HelpText>
+                            If there isn&apos;t enough info on this request,
+                            please get in touch with the requester over email
+                          </HelpText>
+                        </div>
+                        <div className="grid-col-2" />
```

Comment:
Are these empty columns for spacing? This might break down in mobile views. If it does, we should probably use flex or grid to put the space between.

Reply:
I had the same issue myself, and flex or grid did not solve it. I even paired with @sarboc on some of those solutions.

Reply:
Hm interesting. I thought `justify-content: space-between` should take care of it.

It doesn't look like there's an issue with column spacing, but it probably needs to stack once we get to smaller device widths, like maybe stack the radio and email body at around 640px/tablet width?

Reply:
wowow another new css trick!